pkg/e2e/infra/utils.go

```diff
@@ -332,12 +314,12 @@ func getScaleClient() (scale.ScalesGetter, error) {
 
 // nodesSnapShotLogs logs the state of all the nodes in the cluster
 func nodesSnapShotLogs(client runtimeclient.Client) error {
-	nodes, err := getNodes(client)
-	if err != nil {
-		return fmt.Errorf("error calling getNodes: %v", err)
+	nodeList := corev1.NodeList{}
```

Comment:
why this change, we are consistently trying to use get*s abstraction rather than list

Reply:
Put back